package/src/utils/constants.ts

```diff
@@ -1,8 +1,10 @@
+import {Platform} from 'react-native';
+
 export const constants = {
   MARGIN_HORIZONTAL_PADDING: 0.05,
   STEP_NUMBER_TEXT_FONT_SMALL: 8,
   STEP_NUMBER_TEXT_FONT_BIG: 12,
   LIMIT_MIN_VALUE: Number.MIN_SAFE_INTEGER,
   LIMIT_MAX_VALUE: Number.MAX_SAFE_INTEGER,
-  DEFAULT_STEP_RESOLUTION: 100,
+  DEFAULT_STEP_RESOLUTION: Platform.OS === 'android' ? 128 : 1000,
```

Comment:
shouldn't this be 100 not 1000 ?

Reply:
Actually it should be `10000`, but I would stick to `1000` for now, as this is the safest value I was able to observe when testing the solution.